src/cryptonote_core/blockchain.cpp

```diff
@@ -840,6 +841,19 @@ crypto::hash Blockchain::get_block_id_by_height(uint64_t height) const
   }
   return null_hash;
 }
+
+//------------------------------------------------------------------
+void Blockchain::setup_seedhash(const block& b, const uint64_t height) const
+{
+  int cn_variant = b.major_version >= 7 ? b.major_version - 6 : 0;
+  if (cn_variant >= 6) {
```

Comment:
I'd test major_version directly, since this is not a cn variant anymore. Makes things simpler.

Reply:
OK. This was just pasted in from other version tests.